datasets/eth_py150_open/eth_py150_open.py

```diff
@@ -0,0 +1,139 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A redistributable subset of the ETH Py150 corpus"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{kanade2020learning,
+  title={Learning and Evaluating Contextual Embedding of Source Code},
+  author={Kanade, Aditya and Maniatis, Petros and Balakrishnan, Gogul and Shi, Kensen},
+  booktitle={International Conference on Machine Learning},
+  pages={5110--5121},
+  year={2020},
+  organization={PMLR}
+}
+"""
+
+
+_DESCRIPTION = """\
+A redistributable subset of the ETH Py150 corpus, introduced in the ICML 2020 paper 'Learning and Evaluating Contextual Embedding of Source Code'
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://github.com/google-research-datasets/eth_py150_open"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "Apache License, Version 2.0"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://raw.githubusercontent.com/google-research-datasets/eth_py150_open/master/"
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class Ethpy150open(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class EthPy150Open(datasets.GeneratorBasedBuilder):
```
thmis must be the camel case version of the dataset script name `eth_py150_open`